test/fixtures/servers/zookeeper.js

```diff
@@ -7,9 +7,38 @@
 
     var Generic = require('./generic');
     var util = require('util');
+    var exec = require('child_process').exec;
+
+    var service = 'zkServer.sh';
 
     var Zookeeper = function () {
-        Generic.apply(this, ['zkServer.sh', []]);
+        Generic.apply(this, [service, ['start']]);
+        // Zookeeper happily outputs to stderr, so, shut it up
+        this.mute = true;
+
+        var derefStart = this.start;
+
+        this.start = function () {
+            var passargs = arguments;
+            exec('which ' + service, function (error/*, stdout, stderr*/) {
+                // Assume brew osx install in case we didn't find it
```

Comment:
Nice, thank you for adding this check for `zkServer` instead of `zkServer.sh` on brew osx install!